test/run-drun/ok/gc-trigger-acl.drun-run.ok

```diff
@@ -0,0 +1,6 @@
+ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
```

Comment:
I believe the test source file `gc-trigger-acl.mo` is missing in the change set?
Thanks for having created a test case that checks the permissions.